### ISSUE TITLE
Implement basic Webclient as new GUI

### ISSUE THREAD

**Issue by security-companion** [w4af/w4af#254]
We have decided to remove the old GTK-GUI and build upon the api a basic webclient as a GUI.
This issue is about implementing a basic webclient based on React/Vue or other similar frameworks.

**Comment by codders** [comment]
This seems like quite a big ticket - I wonder if it makes sense to split it into smaller parts, and aim for something smaller for the alpha release. I don't mind starting to work slowly on getting a web-based GUI running, but I don't think it should be a blocker.

**Comment by security-companion** [comment]
Yeah, for sure this can be split into several issues and close this one. If you have ideas in mind on how to do this feel free.
This issue can also be moved to the "After alpha-release" milestone.
